tests/message/test_tasks.py

```diff
@@ -348,6 +352,10 @@ def test_size_notint(self):
         with self.assertRaises(exceptions.FieldError):
             helpers.dump_and_load(ttc)
 
+    def test_resources_options(self):
+        ttc = factories.tasks.TaskToComputeFactory()
+        self.assertTrue(hasattr(ttc, 'resources_options'))
```

Comment:
IMHO this method only tests whether `messages.base.Message.load_slots()` works. It initializes all missing slots to `None`. I think it's not necessary to test this in this context.